src/Components/PresentationalComponents/Kebab/KebabItems.js

```diff
@@ -3,22 +3,14 @@ import React from 'react';
 import messages from '../../../Messages';
 import { FormattedMessage } from 'react-intl';
 
-export const kebabItemDownloadJSON = (downloadReport, { ...props }) => (
```

Comment:
Is it correct that `PrimaryToolbar` gives these two export buttons by default? Just checking my understanding :)

Reply:
Yes, historically this was not the case so we had our own buttons, but then our button became obsolette.